cmd/krel/cmd/automate.go

```diff
@@ -0,0 +1,223 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/util/homedir"
+
+	"k8s.io/release/pkg/automate"
+)
+
+const (
+	k8sOrg string = "kubernetes"
+	websiteRepo string = "website"
+)
+
+type automateOptions struct {
+	githubToken string
+	credFile string
+	sheetId string
+	sheetName string
+	rangeCol string
+	colID []string
+	release string
+}
+
+type PrData struct {
+	KubernetesPR int
+	KEP int
+	WebsitePR int
+	Assignee string
+	Sig string
+}
+
+var automateOpts = automateOptions{
+	credFile:  getCredentialFile(),
+	sheetName: "Docs",
+	rangeCol:  "A1:K",
+	colID: []string{"A", "G", "I", "J", "K"},
+}
+
+// announceCmd represents the subcommand for `krel announce`
+var automateCmd = &cobra.Command{
+	Use:           "automate",
```

Comment:
`automate` seems to be a bit too generic. How about `docs`?

Reply:
I think `automate-docs` might be better?
Wdyt?